### DISCUSSION TITLE
Drop support for SSL 1.0 and 1.1

### DISCUSSION THREAD

**Discussion by gustawdaniel** [evertramos/nginx-proxy-automation#374] [Q&A] [answered]
Why?

TLS 1.0 and 1.1 are now officially deprecated due to known security flaws. They have largely disappeared from use across the top one million sites, although a small number of web servers, 0.4 percent, still select one of them during an HTTPS connection.

https://www.f5.com/labs/articles/threat-intelligence/the-2021-tls-telemetry-report

More stats:

https://webtribunal.net/blog/ssl-stats/

On ssl analyser https://www.ssllabs.com/ssltest we getting B instead of A because of support of SSL 1.0 and 1.1 because it simplifies Man in the middle attack.

**Comment by evertramos** [comment, marked as answer]
Hello @gustawdaniel,

Thanks for your message.

This is a discussion for project [nginx-proxy](https://github.com/nginx-proxy/nginx-proxy), which we use in this repo and script.
